catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -3044,10 +3047,18 @@ void MDCCatalogDrawPageControlTile(CGRect frame, NSObject<MDCColorScheme> *color
   CGColorSpaceRelease(colorSpace);
 }
 
-static NSString* const PalettesFontName = @"RobotoMono-Regular";
-
 void MDCCatalogDrawPalettesTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme) {
```

Comment:
Roboto Mono should not be removed from Palettes. It's a design choice for the tile.